site/src/selector.rs

```diff
@@ -33,22 +33,26 @@ use std::fmt;
 use std::ops::RangeInclusive;
 use std::sync::Arc;
 
-pub fn data_for(data: &Index, is_left: bool, query: Bound) -> Option<ArtifactId> {
+/// Gets an artifact id for the given bound
+///
+/// Searches in the indexed commits either from the right or left
+/// If not found in the indexed commits, searches through the artifacts cache
+pub fn data_for(data: &Index, is_left: bool, bound: Bound) -> Option<ArtifactId> {
```

Comment:
This function was hard for me to grok at first. I mean it's simple to understand, but every time I came across it, I'd have to reread the entire body because the name didn't help me at all to remember what it does.

Reply:
As always, happy to make any changes -- I've never had good experiences naming things in rustc-perf.